src/lib/viewers/BaseViewer.js

```diff
@@ -1019,6 +1019,15 @@ class BaseViewer extends EventEmitter {
         return permissions.can_annotate || permissions.can_create_annotations;
     }
 
+    /**
+     * Handles the 'scrolltoannotation' event and calls the annotator scroll method
+     * @param {Object} data
+     * @param {string} data.id Annotation id
+     */
+    handleScrollToAnnotation({ id }) {
+        this.annotator.scrollToAnnotation(id);
```

Comment:
Is this the value that's passed to legacy annotations? It looks like the whole `data` argument is passed down.

Reply:
Ah, yeah it should be passed down for legacy... I'll update that.